### PR TITLE
Shrink node sizes

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description

Our standard DAGs (up to 27 nodes) don't need such amount of resources. We can drop them. Also removing some variables not used in certain platforms and use similar node sizes in all platforms.

### CODE REVIEW

**Comment by amitsagtani97** [comment]
Weird error in the tests - https://github.com/cloud-bulldozer/airflow-kubernetes/runs/7280230516?check_suite_focus=true#step:5:231

`Exception: json file /home/runner/work/airflow-kubernetes/airflow-kubernetes/dags/openshift_nightlies/config/install/alibaba/sdn.json failed to parse`

File: dags/openshift_nightlies/config/install/azure/sdn.json [hunk 1]
```diff
@@ -6,19 +6,19 @@
...
     "openshift_master_vm_size": "Standard_E16s_v3",
     "openshift_worker_vm_size": "Standard_D8s_v3",
-    "openshift_master_root_volume_size": 256,
-    "openshift_worker_root_volume_size": 128,
+    "openshift_master_root_volume_size": 100,
+    "openshift_worker_root_volume_size": 50,
     "openshift_cidr": "10.128.0.0/14",
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_infra_node_vm_size": "Standard_D16s_v3",
+    "openshift_infra_node_vm_size": "Standard_D48s_v3",
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/install/azure/sdn.json:13]: Was this a shrink?

File: dags/openshift_nightlies/config/install/gcp/ovn.json [hunk 2]
```diff
@@ -5,25 +5,16 @@
...
-    "openshift_master_root_volume_type": "io1",
-    "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
-    "openshift_worker_root_volume_type": "gp2",
-    "openshift_worker_root_volume_iops": 0,
+    "openshift_worker_instance_type": "n2-standard-8",
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_infra_node_instance_type": "n1-standard-16",
-    "openshift_workload_node_instance_type": "n1-standard-32",
+    "openshift_infra_node_instance_type": "n2-standard-48",
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/install/gcp/ovn.json:10]: 16->48?

File: dags/openshift_nightlies/config/install/gcp/sdn.json [hunk 3]
```diff
@@ -5,25 +5,16 @@
...
-    "openshift_master_root_volume_type": "io1",
-    "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
-    "openshift_worker_root_volume_type": "gp2",
-    "openshift_worker_root_volume_iops": 0,
+    "openshift_worker_instance_type": "n2-standard-8",
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_infra_node_instance_type": "n1-standard-16",
-    "openshift_workload_node_instance_type": "n1-standard-32",
+    "openshift_infra_node_instance_type": "n2-standard-48",
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/install/gcp/sdn.json:10]: 16->48

File: dags/openshift_nightlies/config/install/aws/sdn.json [hunk 4]
```diff
@@ -10,24 +10,24 @@
...
-    "openshift_master_root_volume_size": 220,
+    "openshift_master_root_volume_size": 100,
     "openshift_master_root_volume_type": "gp3",
     "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
+    "openshift_worker_root_volume_size": 50,
     "openshift_worker_root_volume_type": "gp3",
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "m5.12xlarge",
```
Comment (jtaleric) [inline dags/openshift_nightlies/config/install/aws/sdn.json:20]: Did we want to reduce the instance size here?

**Review by jtaleric — CHANGES_REQUESTED** [review]
Caught a couple increases of instance sizes from 16->48, was that expected?

File: dags/openshift_nightlies/config/install/azure/sdn.json (hunk 1 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/azure/sdn.json:13]: Right, not actually a shrink. Just configured the infra vm size to be in pair with other platforms

File: dags/openshift_nightlies/config/install/aws/sdn.json (hunk 4 above)
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws/sdn.json:20]: I've verified that we're far from using more than 50GiB in our regular 27 nodes tests

File: dags/openshift_nightlies/config/install/aws/ovn.json [hunk 5]
```diff
@@ -10,24 +10,24 @@
...
-    "openshift_master_root_volume_size": 220,
+    "openshift_master_root_volume_size": 100,
     "openshift_master_root_volume_type": "gp3",
     "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
+    "openshift_worker_root_volume_size": 50,
     "openshift_worker_root_volume_type": "gp3",
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "m5.12xlarge",
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply dags/openshift_nightlies/config/install/aws/ovn.json:20]: Yes, the IO intensive infra pods have their own PVs to persist their stuff

**Review by amitsagtani97 — APPROVED** [review]

File: dags/openshift_nightlies/config/install/alibaba/ovn.json [hunk 6]
```diff
@@ -9,29 +9,29 @@
...
     "openshift_master_instance_type": "ecs.r5.4xlarge",
     "openshift_worker_instance_type": "ecs.g6.2xlarge",
-    "openshift_master_root_volume_size": 220,
+    "openshift_master_root_volume_size": 100,
     "openshift_master_root_volume_type": "cloud_essd",
     "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
-    "openshift_worker_root_volume_type": "cloud_essd",
+    "openshift_worker_root_volume_size": 50,
+    "openshift_worker_root_volume_type": "cloud_ssd",
```
Comment (afcollins) [inline dags/openshift_nightlies/config/install/alibaba/ovn.json:20]: cloud_essd -> cloud_ssd is decrease in performance?

**Pull request merged**